include/nbl/builtin/glsl/limits/numeric.glsl

```diff
@@ -69,7 +54,6 @@ float nbl_glsl_numeric_limits_float_epsilon()
 	return nbl_glsl_FLT_EPSILON;
 }
 
-
 float nbl_glsl_ieee754_gamma(float n)
```

Comment:
move all the functions with `ieee754` prefix to the ieee754 glsl header

Reply:
Resolved.